openproblems/tasks/cell_cell_communication/api.py

```diff
@@ -73,8 +80,10 @@ def check_method(adata):
     assert set(adata.uns["ccc_pred"]).issuperset(CCC_PRED_COLUMNS)
     assert "response" not in adata.uns["ccc_pred"]
 
-    assert np.any(np.isin(adata.uns["ccc_pred"]["ligand"].unique(), adata.var.index))
-    assert np.any(np.isin(adata.uns["ccc_pred"]["receptor"].unique(), adata.var.index))
+    assert np.all(np.isin(decomplexify(adata.uns["ccc_pred"]["ligand"].unique()),
```

Comment:
The predicted receptors (mainly) and ligands can be complexes, while the target ligands (at least the ones in our benchmark are cytokines, and hence not complexes). I will change it so that both are "decomplexified", for the sake of completeness.

Reply:
I don't think I completely understand then. Since you're joining the tables, any ligand/receptor that is in the prediction and not in the target will never be "correct", and vice versa -- right? So then they should have exactly the same requirements on target and prediction? Maybe we should talk about this on zoom.

Reply:
Ah, I think I see now. The `ccc_target` we check against the resource (which can contain complexes), while we here we compare the `ccc_pred` to the data features (which don't contain complexes). No need to change anything.

Reply:
Yes, maybe best to talk :)